src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -1085,10 +1085,10 @@ def compute_drift_metrics(
             spikes_in_bin = spikes_in_segment[i0:i1]
             spike_locations_in_bin = spike_locations_in_segment[i0:i1][direction]
 
-            for unit_ind in np.arange(len(unit_ids)):
-                mask = spikes_in_bin["unit_index"] == unit_ind
+            for unit_index, unit_id in enumerate(unit_ids):
```

Comment:
If unit_ids is not the full list the `unit_index` variable is nt a good choici of name and explain the previous mistake.
I would use `i` or `ind`